docs/concepts/policy/restricted-psp.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: 'runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
+spec:
+  privileged: false
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: false
+  # This is redundant with non-root + disallow privilege escalation,
+  # but we can provide it for defense in depth.
+  requiredDropCapabilities:
+    - ALL
```

Comment:
is dropping all capabilities best practice? does that cause problems for most images?

Reply:
See the comment above. Running as non-root drops all capabilities from the effective set (I don't think there's a way around this?) and no_new_privs disables file capabilities, so there is no way to access the bounding set. This is what I meant about it being redundant, but the capabilities might as well be dropped (e.g. if a kernel bug lead to a root privilege escalation, maybe it would still have 0 capabilities).